app/controllers/courses_controller.rb

```diff
@@ -231,7 +231,7 @@ def tag
   def manual_update
     require_super_admin_permissions
     @course = find_course_by_slug(params[:id])
-    UpdateCourseStatsTimeslice.new(@course)
+    UpdateCourseStats.new(@course)
```

Comment:
I'm glad you are making this change. If it's not too much work, this would be better done in a separate PR, but if it's too tedious, I'm okay with keeping it combined with the splitting PR.